__tests__/handleFormSubmit.test.js

```diff
@@ -0,0 +1,49 @@
+import { describe, expect, test, vi, beforeAll } from "vitest";
+import handleFormSubmit from "../src/handleFormSubmit";
+import { ColorList } from "../src/consts";
+
+describe("handleFormSubmit", () => {
+
+  beforeAll(() => {
+    global.chrome = {
+      tabs: {
+        query: () => { }
+      },
+      scripting: {
+        executeScript: () => { }
+      }
+    };
+  });
+
+  test("Should execute script with proper tab id", async () => {
+    const executeScriptSpy = vi.spyOn(global.chrome.scripting, 'executeScript').mockResolvedValue();
```

Comment:
I can see this being used 2 times here, you can move the spy out of the test and use vi.clearAllMocks() either in beforeEach or afterEach

Reply:
My bad